Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -4838,7 +4878,7 @@ def main():  # pragma: no cover
             return_outputs(*get_machine_action_by_id_command(client, args))
 
         elif command == 'microsoft-atp-collect-investigation-package':
-            return_results(get_machine_investigation_package_command(client, args))
+            return_outputs(*get_machine_investigation_package_command(client, args))
```

Comment:
please revert this change,
the new conventions expect to use the `return_results` along with the `CommandResults` object